omnigibson/objects/light_object.py

```diff
@@ -136,6 +137,12 @@ def _initialize(self):
         # Initialize light link
         self._light_link.initialize()
 
+    @property
+    def aabb(self):
+        # This is a virtual object, so no AABB
+        return np.ones(3) * -0.001, np.ones(3) * 0.001
```

Comment:
hmm seems fairly hacky. If you just query the native aabb for a light prim, what does omniverse return? Why not use that value instead?

Reply:
explicitly say omni AABB returns invalid value

Reply:
done